src/less/components/Hendelser.less

```diff
@@ -1,14 +1,46 @@
-.Hendelser {
+.hendelser-content {
+  padding-top: 70px;
+  padding-bottom: 70px;
+}
+
+.infomeldinger-list__container {
+  padding: 1rem 3rem;
+  width: 100%;
+  max-width: 50rem;
+  margin: 0 auto;
+}
+
+.hendelser {
   padding: 4rem 3rem;
   text-align: left;
   width: 100%;
-  max-width: 50rem;
+  max-width: 44rem;
   margin: 0 auto;
-  background: #CCE1F3;
-  border-radius: 15px;
+
+  &__tittel-box {
+    display:flex;
+  }
+
+  &__tittel {
+    padding-right: 10px;
+    padding-bottom: 10px;
```

Comment:
Gjerne bruk rem og em istedenfor pixler i denne fila (gjelder ikke bare disse to linjene), det tilpasser seg bedre zoom o.l.

Reply:
Kult, skal fikse det 😊